tests/ContentServiceTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ContentStore;
+
+use FluentDOM;
+use PHPUnit\Xpath\Assert as XpathAssertions;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use function ob_get_clean;
+use function ob_start;
+
+/**
+ * @backupGlobals enabled
+ */
+final class ContentServiceTest extends KernelTestCase
+{
+    private const ARTICLES_PATH = __DIR__.'/fixtures';
+
+    use XpathAssertions;
+
+    /**
+     * @before
+     */
+    public function setUpFixtures() : void
+    {
+        $_SERVER['ARTICLES_PATH'] = self::ARTICLES_PATH;
```

Comment:
there's not resetting of `$_SERVER`, is it done by PHPUnit automatically?

Reply:
https://github.com/libero/content-store/blob/9fe28b814a5e796befb3e869a372f8a6de9a07ef/tests/ContentServiceTest.php#L16